filters/auth/grantconfig.go

```diff
@@ -371,7 +371,9 @@ func (c *OAuthConfig) RedirectURLs(req *http.Request) (redirect, original string
 	original = u.String()
 
 	u.Path = c.CallbackPath
+	u.RawPath = ""
 	u.RawQuery = ""
+	u.ForceQuery = false
```

Comment:
This is the fix, see https://github.com/golang/go/issues/13488 for details.

Reply:
maybe we should have these comments as comments in code?

Reply:
https://github.com/zalando/skipper/pull/3201#discussion_r1723728995